src/cf.data.pre

```diff
@@ -2304,8 +2304,10 @@ DOC_START
 			Squid, and treat them as unencrypted HTTP messages,
 			becoming the man-in-the-middle.
 
-			The ssl_bump option is required to fully enable
+			The ssl_bump configuration option is required to fully enable
 			bumping of CONNECT requests.
```

Comment:
This paragraph is still very misleading IMO. I suggest the following complete replacement (without formatting): 

* "This option is essentially ignored (with a WARNING) if configuration has no ssl_bump directives."
* "This option is essentially ignored (with a WARNING) unless configuration has ssl_bump directive(s)."